src/Apps/__test__/Fixtures/Order.ts

```diff
@@ -1,4 +1,4 @@
-export const IncompleteOrder = {
+export const UntouchedOrder = {
```

Comment:
Renamed this based on a suggestion from @sweir27 https://github.com/artsy/reaction/pull/1143#discussion_r210924336